frontend/src/features/map/MapContent.tsx

```diff
@@ -0,0 +1,46 @@
+import useInitializeMap from "@/features/map/useInitializeMap";
+import useVisualizeMarker from "@/features/map/useVisualizeMarker";
+import useVisualizeRoute from "@/features/map/useVisualizeRoute";
+import useZoomLevel from "@/features/map/useZoomLevel";
+import ZoomButton from "@/features/map/ZoomButton";
+import { path } from "@/mocks/pathMocks";
+import { css } from "@emotion/react";
+import { useRef } from "react";
+
+declare global {
+  interface Window {
+    kakao: any;
+  }
+}
+
+const MapContent = () => {
+  const mapRef = useRef<HTMLDivElement>(null);
+  const { map } = useInitializeMap({
+    mapRef,
+    centerLat: path[(path.length - 1) / 2].lng,
+    centerLng: path[(path.length - 1) / 2].lat,
```

Comment:
잘 이해했는지 모르겠지만 centerLat이 lng로 나뉘어지고, centerLng가 lat으로 나누어지는 것 같아서 아래와 같이 바꿔야하는게 아닌가 싶습니다 !

그리고 현재 (path.length -1) / 2 는 정수값이 아니게 될 것 같아서  Math.floor 함수로 반올림 하는게 좋을 것 같아요


```suggestion
    const mid = Math.floor(path.length / 2); // 이건 위에서 선언
    centerLat: path[mid].lat,
    centerLng: path[mid].lng,
```

Reply:
해당 연산 부분은 #108 의 유틸 함수로 분리할 예정이어서, 임시로 나누기 2를 진행했었습니다.
예리하시군요... 🤔 👍🏻 
해당 부분은 #108 구현할 때 반영하겠습니다!